VideoLocker/src/main/java/org/edx/mobile/discussion/ThreadBody.java

```diff
@@ -7,7 +7,8 @@
     String topicId;
     String title;
     String rawBody;
-
+    // Automatically follow posts when creating a new post (default behavior on desktop) [MA-1380]
```

Comment:
Yeah, JIRA links are useful for TODOs, but not necessary to justify every change. If there is a super long JIRA conversation that would be hard to summarize in a comment, it might also make sense to just link it, but note that not everyone working with the code will have access to all of our tickets